documentation/book/topic-operator.adoc

```diff
@@ -1,47 +1,54 @@
-[id='topic-operator-{context}']
-== Topic Operator
+// This assembly is included in the following assemblies:
+//
+// master.adoc
 
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
 
-include::con-what-the-topic-operator-does.adoc[leveloffset=+2]
+[id='topic-operator-{context}']
+= Topic Operator
 
-include::con-how-the-topic-operator-works.adoc[leveloffset=+2]
+:context: str
+include::con-what-the-topic-operator-does.adoc[leveloffset=+1]
 
-=== Usage Recommendations
+:context: str
+include::con-how-the-topic-operator-works.adoc[leveloffset=+1]
 
-. Either always operate on `KafkaTopic` resources or always operate directly on topics. That is, avoid routinely using both methods for a given topic.
-. When creating a `KafkaTopic` resource:
-    * Remember that the name cannot be changed later.
-    * Choose a name for the `KafkaTopic` resource that reflects the name of the topic it describes.
-    * Ideally the `KafkaTopic.metadata.name` should be the same as its `spec.topicName`.
-      To do this, the topic name will have to be a https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name].
-. When creating a topic:
-    * Remember that the name cannot be changed later.
-    * It is best to use a name that is a https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name],
-      otherwise the operator will have to modify the name when creating
-      the corresponding `KafkaTopic`.
+:context: str
+include::ref-topic-operator-usage-recommendation.adoc[leveloffset=+1]
 
 [id='format-of-the-kafkatopic-{context}']
-=== Format of the `KafkaTopic`
+== Format of the `KafkaTopic`
 
 For detailed reference of the API for `KafkaTopic` see xref:type-KafkaTopic[`KafkaTopic` API documentation].
 
-include::proc-creating-a-topic.adoc[leveloffset=+2]
+:context: str
+include::proc-creating-a-topic.adoc[leveloffset=+1]
 
-include::proc-changing-a-topic.adoc[leveloffset=+2]
+:context: str
+include::proc-changing-a-topic.adoc[leveloffset=+1]
 
-include::proc-deleting-a-topic.adoc[leveloffset=+2]
+:context: str
+include::proc-deleting-a-topic.adoc[leveloffset=+1]
 
-include::ref-topic-operator-environment.adoc[leveloffset=+2]
+:context: str
+include::ref-topic-operator-environment.adoc[leveloffset=+1]
 
-=== Resource limits and requests
+== Resource limits and requests
 
 The Topic Operator can run with resource limits:
 
 * When it is deployed by the Cluster Operator these can be specified in the `Kafka.spec.topicOperator.resources` property.
 * When it is not deployed by the Cluster Operator these can be specified on the `Deployment` in the usual way.
 
-==== Minimum Resource Requirements
+=== Minimum Resource Requirements
```

Comment:
I've mentioned this before, but I think a reference covering the minimum resource requirements of all the components would be a better place to put this. It also needs validation from testing and a way to maintain the correct numbers.